apps/bestofjs-nextjs/src/app/featured/page.tsx

```diff
@@ -0,0 +1,91 @@
+import { StarIcon } from "@/components/core";
+import { PageHeading } from "@/components/core/typography";
+import {
+  ProjectPageSearchParams,
+  parseSearchParams,
+  stateToQueryString,
+} from "@/components/project-list/navigation-state";
+import { ProjectPaginatedList } from "@/components/project-list/project-paginated-list";
+import {
+  SortOption,
+  SortOptionKey,
+  sortOrderOptionsByKey,
+} from "@/components/project-list/sort-order-options";
+
+import { searchClient } from "../backend";
+import { getFeaturedProjectsRequest } from "../backend-search-requests";
+import { ProjectSearchQuery, SearchQueryUpdater } from "../projects/types";
+
+type PageProps = {
+  searchParams: ProjectPageSearchParams;
+};
+
+export default async function FeaturedProjectsPage({
+  searchParams,
+}: PageProps) {
+  const searchState = parseSearchParams(searchParams, { sort: "newest" });
+  const { projects, total } = await fetchFeaturedProjects(searchState);
+  const { page, limit, sort } = searchState;
+
+  const buildPageURL = (updater: SearchQueryUpdater<ProjectSearchQuery>) => {
+    const nextState = updater(searchState);
+    const queryString = stateToQueryString(nextState);
+    return "/featured?" + queryString;
+  };
+
+  return (
+    <>
+      <PageHeading
+        icon={<StarIcon size={36} />}
+        title={<>Featured Projects</>}
+        subtitle={
+          <>An arbitrary selection of important projects with distinct logos.</>
```

Comment:
This isn't used on the page right? I cannot find it.

The wording is a bit excessive.

Reply:
It's part of the "featured" page mentioned in the PR's description, it's the new page I've added.

Open to any suggestion for the wording, we don't need to mention the fact the projects have custom logos indeed (the SVG files hosted on our domain, instead of the usual GitHub avatar, FYI)